modules/osd-rhoam.adoc

```diff
@@ -1,11 +1,24 @@
+// Module included in the following assemblies:
+//
+// * adding_service_cluster/available-services.adoc
+// * adding_service_cluster/rosa-available-services.adoc
 
 [id="osd-rhoam_{context}"]
-
 = Red Hat OpenShift API Management
 
+The Red Hat OpenShift API Management (OpenShift API Management)  service is available as an add-on to your {product-title} on AWS cluster. OpenShift API Management is a managed API traffic control and API program management solution that is based on the 3scale API Management platform and also includes an implementation of the Red Hat Single Sign-On to secure and protect your APIs.
+
+This free OpenShift API Management entitlement tier provides:
```

Comment:
Do we need _free_ here? Seems like marketing-speak.

Reply:
I believe this is actually the purpose of the original BZ. It specifies that there's no costs or limits: https://issues.redhat.com/browse/OSDOCS-2403.

Anyway we can word this better?